File: Doubao_MarsCode/java/MC0048/Main.java
package MC0048;

/**
 * 48. a替换函数
 */
public class Main {
    public static String solution(String s) {
        return s.replaceAll("a", "%100");
    }

    public static void main(String[] args) {
        System.out.println(solution("abcdwa").equals("%100bcdw%100"));
        System.out.println(solution("banana").equals("b%100n%100n%100"));
        System.out.println(solution("apple").equals("%100pple"));
    }
}

File: Doubao_MarsCode/go/MC0048/main.go
package main

import (
	"fmt"
	"strings"
)

func solution(s string) string {
	return strings.ReplaceAll(s, "a", "%100")
}

func main() {
	fmt.Println(solution("abcdwa") == "%100bcdw%100")
	fmt.Println(solution("banana") == "b%100n%100n%100")
	fmt.Println(solution("apple") == "%100pple")
}
